pyproject.toml

```diff
@@ -1,7 +1,80 @@
 [build-system]
-requires = [ "setuptools", "wheel" ]
+requires = ["setuptools >= 63.0.0"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "bitsandbytes"
+dynamic = ["version"]
+description = "k-bit optimizers and matrix multiplication routines."
+authors = [{name="Tim Dettmers", email="dettmers@cs.washington.edu"}]
+requires-python = ">=3.8"
```

Comment:
We don't actually build/test for 3.8 in the workflow. Should we specify 3.9 as a minimum version, or verify for 3.8 in the workflow?

Reply:
It should still work work 3.8, and we even have `ruff` targeting for that too. So that is why I submitted #1076, based also on https://github.com/TimDettmers/bitsandbytes/discussions/990#discussioncomment-8558070. So agreed: 3.8 should probably be validated in CI or dropped. I would like to make 3.9 the minimum personally, but not sure what everyone else thinks?

Reply:
Ah, yes. Better to target 3.8 in tests until decided to drop it